src/encoded/schemas/ivp_c2.json

```diff
@@ -0,0 +1,1150 @@
+                        "1 Yes (CONTINUE WITH QUESTION 1b)"
+                    ]
+                },
+                "mocareas": {
+                    "title": "1a1. Was any part of MoCA administered?  If no, enter reason code, 95-98)",
+                    "type": "string",
+                    "enum": [
+                        "95 = Physical problem",
+                        "96 = Cognitive/behavior problem",
+                        "97 = Other problem",
+                        "98 = Verbal refusal"
+                    ]
+                },
+                "mocaloc": {
+                    "title": "1b. MoCA was administered:",
+                    "type": "string",
+                    "enum": [
+                        "1 In ADC or clinic",
+                        "2 In home",
+                        "3 In person - other"
+                    ]
+                },
+                "mocalan": {
+                    "title": "1c. Language of MoCA administration:",
+                    "type": "string",
+                    "enum": [
+                        "1 English",
+                        "2 Spanish",
+                        "3 Other(SPECIFY BELOW):"
+                    ]
+                },
+                "mocalanx": {
+                    "title": "1c1. Specify language of MoCA administration",
+                    "type": "string"
+                },
+                "mocavis": {
+                    "title": "1d. Subject was unable to complete one or more sections due to visual impairment: ",
+                    "type": "string",
+                    "enum": [
+                        "0 No",
+                        "1 Yes"
+                    ]
+                },
+                "mocahear": {
+                    "title": "1e. Subject was unable to complete one or more sections due to hearing impairment: ",
+                    "type": "string",
+                    "enum": [
+                        "0 No",
+                        "1 Yes"
+                    ]
+                },
+                "mocatots": {
+                    "title": "1f. TOTAL RAW SCORE - UNCORRECTED (Not corrected for education or visual/hearing impairment)&nbsp ",
+                    "description": "((0-30, 88))(Enter 88 if any of the following MoCA items were not administered:  1g-1l, 1n-1t, 1w-1bb)",
+                    "type": "string",
+                  "pattern":"^[1-2]?[0-9]$|^30$|^88$"
+                },
+                "alert_mocatots": {
+                    "title": "WARNING - a value of 88 should be entered in the box above.  At least one of the following MoCA items was not administered: 1g-1l, 1n-1t, 1w-1bb.",
+                    "type": "string"
+                },
+                "mocatrai": {
+                    "title": "1g. Visuospatial/executive -- Trails",
+                    "description": "((0-1, 95-98))",
+                    "type": "string",
+                    "pattern":"^[0-1]$|^95$|^96$|^97$|^98$"
```

Comment:
use anyof to include pattern for 1 -2 and enum for :
95, 95 = Physical problem
96, 96 = Cognitive/behavior problem
97, 97 = Other problem
98, 98 = Verbal refusal

So we won't lose readability.

Reply:
1. This is a text box input number only, the meanings are already showed in the form instruction: see below.
![image](https://user-images.githubusercontent.com/38848214/100898683-a301d380-3486-11eb-823c-8e2a27498b71.png)
![image](https://user-images.githubusercontent.com/38848214/100898745-b2811c80-3486-11eb-9364-540ca730a43c.png)

2. I can add each property description with these interpretation, if we want.  

3.  If I change to  "pattern":"^[0-1]$|^95 = Physical problem$|^96 = Cognitive\/behavior problem$|^97 = Other problem$|^98 = Verbal refusal$", will the mapping still work?